R/use_ensemble_criteria.R

```diff
@@ -36,7 +39,8 @@ use_ensemble_criteria <- function(forecasts,
                                   exclude_models = NULL,
                                   exclude_designated_other = TRUE,
                                   return_criteria = TRUE,
-                                  eval_dir = here::here("evaluation", "weekly-summary")) {
+                                  eval_dir = here::here("evaluation", "weekly-summary"),
+                                  rel_wis_cutoff = Inf) {
```

Comment:
What happens is your cutoff is so low that you remove all models? Do you get a helpful error message?

Reply:
A warning, now.

Reply:
I'll leave error processing to downstream use, just in case removing all models is OK for whatever purpose it's being used.